juju/juju
# Extract common agent related functionality from juju-updateseries

## Description of change
This PR address relocation of functions which are common in update series and also agent specific functions from service folder.
## QA steps
This change is required to make the code more compact and reduce redundant code.
verify the execution of bootstrap, deploy and update series command as well.
## Documentation changes
N/A
## Bug reference
https://bugs.launchpad.net/juju/+bug/1634390
